tests/service-catalog/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN CGO_ENABLED=0 go build -o /env-tester.bin ./cmd/env-tester/main.go
 RUN CGO_ENABLED=0 go test -tags=acceptance -c ./test -o /servicecatalog.test
 
 FROM alpine:3.10
```

Comment:
why it's not migrated to scratch? if cannot be migrated then add comment explaining why

Reply:
added a comment